charts/valtimo-backend/templates/deployment.yaml

```diff
@@ -58,6 +58,17 @@ spec:
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
       {{- with .Values.nodeSelector }}
```

Comment:
This is on the wrong line, should be moved right before the `nodeSelector` part.